src/addrman.cpp

```diff
@@ -101,26 +103,13 @@ double CAddrInfo::GetChance(int64_t nNow) const
     return fChance;
 }
 
-CAddrMan::CAddrMan(std::vector<bool> asmap, bool deterministic, int32_t consistency_check_ratio)
-    : insecure_rand{deterministic}
-    , nKey{deterministic ? uint256{1} : insecure_rand.rand256()}
-    , m_consistency_check_ratio{consistency_check_ratio}
-    , m_asmap{std::move(asmap)}
-{
-    for (auto& bucket : vvNew) {
-        for (auto& entry : bucket) {
-            entry = -1;
-        }
-    }
-    for (auto& bucket : vvTried) {
-        for (auto& entry : bucket) {
-            entry = -1;
-        }
-    }
-}
+AddrMan::AddrMan(std::vector<bool> asmap, bool deterministic, int32_t consistency_check_ratio)
```

Comment:
Could these `AddrMan` ctor and dtor methods also be moved to the bottom, below the `AddrManImpl` methods?

Reply:
done